packages/boxel/addon/components/boxel/card-container/index.css

```diff
@@ -2,13 +2,22 @@
   position: relative;
   background-color: var(--boxel-light);
   border-radius: var(--boxel-border-radius);
-  overflow: hidden;
```

Comment:
In case the dropdown content is too big to be fitted inside the container edge, `overflow: hidden;` will cut it off:

![image](https://user-images.githubusercontent.com/273660/133427553-5407a81e-1622-417b-9e64-f1f1ed13198f.png)

Removing `overflow: hidden;` fixes this issue):

![image](https://user-images.githubusercontent.com/273660/133427643-3da6ebb2-cd32-4032-9039-ea955d238bc5.png)

However, removing that produces a new bug - the rounded corners of the card container stop being round 🤯:

![image](https://user-images.githubusercontent.com/273660/133428043-bd0782a7-b87c-4c84-adbd-de9540a0b3f3.png)

I think `overflow: hidden;` was here to make the border radius work when there are children elements present, [as discussed here](https://stackoverflow.com/questions/10995294/border-radius-not-working).

So to support both elements that need to span out of the container (dropdowns etc), and keeping the border radius, I added the `.boxel-card-container > *:first-child` and `.boxel-card-container > *:last-child` rules for border radius.

Reply:
We use `CardContainer` very liberally at the moment - not just as a workflow card but as anything that needs the border radius + background + shadow, `first-child` might not always refer to the header, `last-child` might not always refer to the chin; likewise they may not be attached to the top/bottom. What do you think about making these changes in the `CardPay::ActionCardContainer` component instead?

Reply:
Shouldn't dropdowns be doing something to remove themselves from DOM hierarchy to avoid this?

Reply:
@Aierie thanks, I think you're right - better to put this in `.action-card-container` then!

@lukemelia Yeah I think that's the default behaviour for power select, however we use `@renderInPlace={{true}}`, which renders it right there in the component. If I remove this, I get an odd error:

![image](https://user-images.githubusercontent.com/273660/133579248-3f01a77b-7c68-4ffb-aad6-12f73a513018.png)

It looks like we're using `@renderInPlace={{true}}` in quite a few other places as well, but I'm not sure why 🤔

Reply:
> @Aierie thanks, I think you're right - better to put this in `.action-card-container` then!
> 
> @lukemelia Yeah I think that's the default behaviour for power select, however we use `@renderInPlace={{true}}`, which renders it right there in the component. If I remove this, I get an odd error:
> 
> ![image](https://user-images.githubusercontent.com/273660/133579248-3f01a77b-7c68-4ffb-aad6-12f73a513018.png)
> 
> It looks like we're using `@renderInPlace={{true}}` in quite a few other places as well, but I'm not sure why 🤔

I've dealt with this error as well and `@renderInPlace={{true}}` was the workaround. I tried the suggestions in this thread, which stopped the error message but then the dropdown did not appear anywhere on the UI: https://discuss.emberjs.com/t/console-error-when-making-addon-component/18360/16 This one recommends going back to an older version: https://github.com/cibernox/ember-basic-dropdown/issues/614. 

There's a card in Merchant Creation workflow that also had this dropdown issue (where the user selects a prepaid card from the dropdown). I had removed overflow for that specific card. We should only be using the CardContainer for isolated and embedded cards, but the header and footer are sometimes optional or hidden in various card modes so let's keep an eye on overflow and card borders.

Reply:
Got it. Thanks for chiming in.